src/battle_ai_util.c

```diff
@@ -2096,7 +2088,7 @@ bool32 HasMoveWithLowAccuracy(u32 battlerAtk, u32 battlerDef, u32 accCheck, bool
             if (ignoreStatus && IS_MOVE_STATUS(moves[i]))
                 continue;
             else if ((!IS_MOVE_STATUS(moves[i]) && gMovesInfo[moves[i]].accuracy == 0)
-              || AI_GetBattlerMoveTargetType(battlerAtk, moves[i]) & (MOVE_TARGET_USER | MOVE_TARGET_OPPONENTS_FIELD))
+                  || GetBattlerMoveTargetType(battlerAtk, moves[i]) & (MOVE_TARGET_USER | MOVE_TARGET_OPPONENTS_FIELD))
```

Comment:
Is this spacing intended to have been changed? I understand there are a few different standards of style among contributors, just ensuring this is intended before merging. Looks great to me otherwise.

Reply:
Since the the expansion uses those following formats for the majority of the codebase I try to stay consistent with it if I change code. I personally like the first option so that is what I changed it to.
```
if (this
 || that
 || these
 || those)
{
    // stuff
}
```
```
if (this
    || that
    || these
    || those)
{
    // stuff
}
```